QgisModelBaker/libqgsprojectgen/generator/generator.py

```diff
@@ -101,12 +101,19 @@ def layers(self, filter_layer_list=[]):
             if filter_layer_list and record['tablename'] not in filter_layer_list:
                 continue
 
-            alias = record['table_alias'] if 'table_alias' in record else ''
             is_domain = record['kind_settings'] == 'ENUM' or record[
                 'kind_settings'] == 'CATALOGUE' if 'kind_settings' in record else False
             is_structure = record['kind_settings'] == 'STRUCTURE' if 'kind_settings' in record else False
             is_nmrel = record['kind_settings'] == 'ASSOCIATION' if 'kind_settings' in record else False
 
+            alias = record['table_alias'] if 'table_alias' in record else ''
```

Comment:
```suggestion
            alias = record['table_alias'] if 'table_alias' in record else None
```

Maybe? I always tend to prefer explicit "no information" over an empty string -- except where not possible.

Reply:
Actually, wouldn't that be much more readable?

```suggestion
            if 'table_alias' in record:
                alias = record['table_alias']
            else:
```

Reply:
On the second one I disagree. On such small if-clause I think even if the if-clause itself is slightly more readable, it goes on costs of the readability of the whole function, since it makes it bigger with a lot of simple ifs.

Reply:
Might just be my general aversion against ternary operators :-)

Reply:
Yeah, I generally like the if then else way better as well. But for this case it's okay.